src/mesh.c

```
@@ -389,583 +368,295 @@ int mesh_update_routing(uint16_t source, uint16_t from_node, int16_t rssi, uint8
-{
-    if (node_id == 0 || node_id == mesh_state.our_node_id)
-    {
-        return -1;
-    }
+/* ============================================================================
+ * Radio callback glue
+ * ============================================================================ */
 
-    /* Find existing neighbor */
-    int idx = mesh_find_neighbor_index(node_id);
+static void mesh_rx_handler(uint8_t* data, size_t len, int16_t rssi, int8_t snr) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'mesh_rx_handler' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)